### ISSUE TITLE
WCAG/HTML name computation discrepancy with `<img alt="" title="foo">

### ISSUE THREAD

**Issue by cookiecrook** [w3c/html-aam#99]
The following text in HTML-AAM:
> ~If "any labeling attribute … specifies an empty label … indicates the author's intention to indicate that the img is decorative or redundant. In this case, the user agent MUST set the name to an empty string."

Conflicts with WCAG Success Criteria H67: https://www.w3.org/TR/WCAG20-TECHS/H67.html
> For each image that should be ignored: Check that title attribute is either absent or empty. Check that alt attribute is present and empty. ... 

Furthermore, both Blink and Gecko allow this behavior. WebKit currently matches the HTML-AAM spec, but will likely change to match the other behavior. https://webkit.org/b/173870

Test case 7 here:
https://bugs.webkit.org/attachment.cgi?id=314203

**Comment by cookiecrook** [comment]
HTML-AAM also specifies this normative requirement in an non-normative note. The normative text does not include this "empty label" exception.

**Comment by cookiecrook** [comment]
Similar issue in https://github.com/w3c/aria/issues/602

**Comment by stevefaulkner** [comment]
I think this is a reasonable change, as while it is a conformance error to use the empty alt, title with content pattern, it is useful for end users to get some accessible name in this case.

**Comment by stevefaulkner** [comment]
I have removed the 'normative note' ;-) which should resolve the issue from the aam perspective.
